### ISSUE TITLE
Publish 5.x version to include the license file(s)?

### ISSUE THREAD

**Issue by erickt** [notify-rs/notify#460]
Hello all,

I noticed that the 5.0.0 notify crate does not include a license file. Would it be possible to release a point release to include the actual license text in the crate?

Thanks, that'll help make it easier to use.

**Comment by 0xpr03** [comment]
What do I have to do for that ? Because I didn't change anything in the project regarding the license since pre 5.0, so it being missing means cargo didn't include it ?

**Comment by erickt** [comment]
I think this happened when notify shifted over to using workspaces. By default it'll pick up the license files where you ran it, but I don't think this happens by default with workspaces. In my projects, I just symlink to the loplevel license, with something like this:

```
% cd notify
% ln -s ../LICENSE
% ln -s ../LICENSE.ARTISTIC
% cd ../notify-debouncer-mini
% ln -s ../LICENSE
% ln -s ../LICENSE.ARTISTIC
```

You can then use `cargo package` and look in the `targets/package` to make sure the files are included.

I wasn't sure what's covered by the CC0 or artistic license, so I didn't upload a PR for this.

**Comment by JohnTitor** [comment]
Yeah, my some projects, e.g. [actix-web](https://github.com/actix/actix-web/tree/master/actix-web) also use symbolic links to include license files. 

> I wasn't sure what's covered by the CC0 or artistic license, so I didn't upload a PR for this.

I'm also not sure, but there's no reason not to do that. I've submitted #461.

**Comment by 0xpr03** [comment]
Yeah in hindsight it makes total sense that it's not in the actual workspace crate anymore.. Thanks!

**Comment by 0xpr03** [comment]
Leaving this open until I've done the release

**Comment by 0xpr03** [comment]
#464

**Comment by 0xpr03** [comment]
Closed by 85a231f01db35f9606b68fbcfc4d046c3b430986
